tsconfig.test.json

```diff
@@ -0,0 +1,21 @@
+{
+    "compilerOptions": {
```

Comment:
rather than redeclaring all of this, I would use the [`extends` keyword](https://www.typescriptlang.org/tsconfig#extends)

Reply:
done